### ISSUE TITLE
github exporter: show tag within release in github_repo_release_downloads

### ISSUE THREAD

**Issue by 532910** [grafana/agent#6495]
### Request

`github_repo_release_downloads` gives only `release` which could be a human name. Please add support to return tag name in addition to release.

### Use case

get the latest version

**Comment by tpaschalis** [comment]
I agree this might be useful. Like in #6494, I think it's best if we manage to get an upstream fix that can benefit the entire community.

I'm hoping that the github_exporter project finds a new home (https://github.com/prometheus-community/community/issues/44) and it continues working.

**Comment by hainenber** [comment]
The github-exporter already moved to https://github.com/githubexporter, from its previous org infinityworks :D

I'll open a upstream PR for this

**Comment by hainenber** [comment]
Made an upstream PR to address this issue. Hopefully the authors taking a look :D

**Comment by 532910** [comment]
I just tried to get the latest release with the current implementation and really didn't find a way to get it:
`github_repo_release_downloads{repo="some-repo"})` gives several time-series with different `release` labels, but how to get the latest one?

I believe the PR above should be applied, but really it doesn't fixes this issue fully. I will be nice to have a separate metric to track the latest release:
```
% curl -s https://api.github.com/repos/grafana/agent/releases/latest | jq .tag_name 
"v0.40.1"
```

**Comment by hainenber** [comment]
If your target is to track the latest release, [GH doc](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release) recommends to sort releases by their `created_at` attribute as the timestamp is the date of the commit used for the release.

Since this is considered to be useful by other folks, I'll keep the upstream PR open.

**Comment by 532910** [comment]
1. GH doc does not recommend to sort releases by their created_at attribute! It just explains the latest release as:
> the most recent non-prerelease, non-draft release, sorted by the created_at attribute.
GH doc recommends the same I showed with curl:
```
GET /repos/{owner}/{repo}/releases/latest
```

2. GH doc has nothing common with how to get the latest release from github_exporter.

**Comment by hainenber** [comment]
I think my wording before, `recommends`, was too far-reaching but honestly, the addition of one more label `is_latest` or some sort to indicate a tag to be latest would be pricey, API-wise.

The `ListRelease` API response doesn't have any attribute to indicate a release to be _latest_. That requires another call to GH using the API you've described. I'm not sure the upstream author would agree with this change, btw.
